DataFormats/TrackReco/interface/HitPattern.h

```diff
@@ -205,6 +206,12 @@ class HitPattern
     /// RPC region: 0 = barrel, 1 = endcap. Only valid for muon RPC patterns, of course.
     static uint16_t getRPCregion(uint16_t pattern);
 
+    /// GEM roll: (1-8). Only valid for muon GEM patterns, of course.
+    static uint16_t getGEMRoll(uint16_t pattern);
```

Comment:
(see also comments below) packing of GEMRoll number will not work for GE2/1 (more than 8 rolls).
There are only 3 bits available and it looks like they are better be spent on the station index than on the roll.